src/sas/readstat_sas7bcat_read.c

```diff
@@ -133,6 +134,9 @@ static readstat_error_t sas7bcat_parse_value_labels(const char *value_start, siz
     }
 
 cleanup:
+    if(label)
+        free(label);
```

Comment:
It's not important, but for future reference: `free` will accept a NULL pointer, so it's not necessary to check it with `if(label)`.